pkg/operator/controller/ingress/deployment.go

```diff
@@ -498,6 +514,11 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 		env = append(env, corev1.EnvVar{Name: WildcardRouteAdmissionPolicy, Value: "false"})
 	}
 
+	if HTTP2IsDisabledByAnnotation(ci.Annotations) || HTTP2IsDisabledByAnnotation(ingressConfig.Annotations) {
+		log.Info("Disabling HTTP/2 for router deployment", "namespace", deployment.Namespace, "name", deployment.Name)
```

Comment:
This will get logged every time the operator reconciles the ingresscontroller, which seems excessive.  I'd just as soon drop the log statement.